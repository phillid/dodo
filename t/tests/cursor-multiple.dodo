u
c6
c25
c44
e/uiff /
w/eggle/

u
c13
c32
c51
e/eggle/
w/apple/
