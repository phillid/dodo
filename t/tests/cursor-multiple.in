fleg quiff sneggle
blaf quiff kleggle
qnef puiff pleggle
