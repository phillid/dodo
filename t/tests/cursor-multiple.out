fleg qegglesnapple
blaf qeggleklapple
qnef peggleplapple
